workshops/sprig/README.md

```diff
@@ -16,40 +16,52 @@ locales: 'es-xl, pt-br'
 > If you are a solo hacker you can jump straight into the [Sprig editor](https://editor.sprig.hackclub.com).
 > You may be able to pick up some helpful tips reading through this though.
 
-<img width="500" alt="Screen Shot 2022-09-23 at 4 41 25 PM" src="https://user-images.githubusercontent.com/27078897/192053241-51cd9891-2e13-482c-b5e0-2d285b87f62c.png">
+<img width="500" alt="Screen Shot 2023-03-03 at 5 48 32 PM" src="https://cloud-guxu865fy-hack-club-bot.vercel.app/0image.png">
 
 We designed Sprig to accomodate workshops with minimal instruction and maximal making and exploring. This is how I would run Sprig as a workshop.
 
 ## Demo (2 minutes)
 
 Give a quick demonstration of a fairly complete game with accessible code. You can go to the [gallery](https://sprig.hackclub.com/gallery) and sort by the `beginner` tag.
 
-<img width="500" alt="Screen Shot 2022-09-23 at 5 00 24 PM" src="https://user-images.githubusercontent.com/27078897/192056026-c4c5492d-00e8-44fb-be35-6b74a1c3bfe2.png">
+<img width="500" alt="Screen Shot 2023-03-03 at 6 14 44 PM" src="https://cloud-9h10p1the-hack-club-bot.vercel.app/0image.png">
 
 Here are a few games that meet the criteria:
 
 - [Sokoban](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/sokoban.js)
 - [Penguin Slide](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/penguin_slide.js)
 - [Nomis](https://editor.sprig.hackclub.com/?file=https://raw.githubusercontent.com/hackclub/sprig/main/games/nomis.js)
 
+## Account (2 minutes)
+
+For that you and your club don't lose your files, we use an account system, here is the step by step:
+
+1. Click on the "Log in" button at the top right of the page
+<img width="500" alt="Screen Shot 2023-03-03 at 6 35 57 PM" src="https://cloud-c0fezx07u-hack-club-bot.vercel.app/0image.png">
+
+2. Fill in the field with an email and then click on the "Send code" button
+<img width="500" alt="Screen Shot 2023-03-03 at 6 40 41 PM" src="https://cloud-5tajrvzec-hack-club-bot.vercel.app/0image.png">
+
+3. You will receive an email like this, then just put this code in the field "Code:"
+<img width="500" alt="Screen Shot 2023-03-03 at 6 46 27 PM" src="https://cloud-3e970nukr-hack-club-bot.vercel.app/0image.png">
+
+That's it! Your account is ready!
+
```

Comment:
I'm not sure if we should have the user go through the log in page in the tutorial, perhaps we should encourage doing the save-to-email-normally thing instead. Thoughts @leomcelroy?

Reply:
On second thought, this is probably fine, but perhaps we should include a note that it's optional.

Reply:
thinking well, this can take too much time with a bad wifi, as an alternative, we can take and put it on the `my_first_game` workshop